src/provider/application.js

```diff
@@ -72,7 +73,18 @@ class Application extends EventEmitter {
       const width = calculateWidth(this.resizeConfig.WidthCalculationMethod);
       this.JSONRPC.notification('resize', [null, `${width}px`]);
     } else {
-      const height = calculateHeight(this.resizeConfig.heightCalculationMethod);
+      let height = calculateHeight(this.resizeConfig.heightCalculationMethod);
+
+      if (this.targetNodes.length > 0) {
+        const maxTargetHeight = this.targetNodes
+          .map(selector => document.querySelector(selector))
+          .filter(node => node)
+          .map(node => getOffsetToBody(node) + node.offsetHeight)
+          .reduce((max, curr) => Math.max(max, curr), 0);
+
+        height = Math.max(height, maxTargetHeight);
```

Comment:
I think you can remove the last reduce above and assign the array of node heights to `nodeHeights` and do:

```js
nodeHeights.push(offsetHeight);
return Math.max(...nodeHeights);
```